components/TaskItem.tsx

```diff
@@ -27,18 +27,31 @@ export default function TaskItem({ task }) {
   }
 
   return (
-    <Pressable
-      style={({ pressed }) => [styles.itemContainer, pressed && styles.pressed]}
-      onPress={toggle}
-    >
-      <View className="mx-2">{icon}</View>
-
-      <View className="flex-1 flex-row ml-2 text-left">
-        <Text className=" font-bold">
-          {task.content} - {task.category}
+    <View className="flex-1 flex-row justify-between">
+      <Pressable
+        style={({ pressed }) => [
+          styles.itemContainer,
+          pressed && styles.pressed,
+        ]}
+        onPress={toggle}
+      >
+        <View className="mx-2">{icon}</View>
```

Comment:
Instead of reasiging a componet to variable (`let icon = ...`)

Use ternary operator:

```TSX
<>
{ 
    done ? <AntDesign name="checksquareo" size={30} color="black" /> : <Feather name="square" size={30} color="black" />;
}
</>
```